plugin/core/restore_lines.py

```diff
@@ -7,15 +7,16 @@ def __init__(self):
         self.saved_lines = []  # type: List[dict]
 
     def save_lines(self, locations: List[int], view: sublime.View) -> None:
-        # Clear previously saved lines
-        self.clear()
+        change_id = view.change_id()
 
         for point in locations:
-            text = view.substr(view.line(point))
-            row, _col = view.rowcol(point)
+            line = view.line(point)
+            change_region = (line.begin(), line.end())
+            text = view.substr(line)
```

Comment:
Anything which prevents the `line` to be stored directly? Doing so would avoid converting `region->tuple->region`.

Reply:
Regions can't be serialized for transmission in `run_command`, which is required for the commit completion command.

Reply:
Guess the python 3.3 version of `sublime.Region` does not support iteration. This will be available as soon as we can move to python 3.8

Reply:
I didn't know you could do that `tuple(region)` with a Region. Cool :)

Reply:
'Region' object is not iterable. 
```
    saved_line['change_region'] = tuple(saved_line['change_region'])
TypeError: 'Region' object is not iterable
```
Looks like the LSP plugin is still using python 3.3.